examples/frameworks/strands_demo/bedrock_agentcore/scripts/update_nat.py

```
@@ -0,0 +1,17 @@
+
+response = client.update_agent_runtime(agentRuntimeId='<AGENT_RUNTIME_ID>',
+                                       agentRuntimeArtifact={
+                                           'containerConfiguration': {
+                                               'containerUri':
+                                                   '<AWS_ACCOUNT_ID>.dkr.ecr.<AWS_REGION>.amazonaws.com/strands-demo:latest'
+                                           }
+                                       },
+                                       networkConfiguration={"networkMode": "PUBLIC"},
+                                       roleArn='<IAM_AGENTCORE_ROLE>')
+
+print(f"Agent Runtime created successfully!")
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)